firmware/release-notes.md

```diff
@@ -1,7 +1,20 @@
 # Raspberry Pi4 bootloader EEPROM release notes
 
 ## 2022-02-04 - Network Install - BETA
-   * New network install feature for the bootloader
+   * New network install feature for the bootloader. To disable network install
+     (e.g. in an industrial product) set NET_INSTALL_ENABLED=0 in the EEPROM
+     config or HDMI_DISABLE=1.
+     Network install is not enabled in secure-boot mode.
```

Comment:
I don't think it's disabled for secure-boot at the moment

Reply:
Ah, good point. It just won't work :)